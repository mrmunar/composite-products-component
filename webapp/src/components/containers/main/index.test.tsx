import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Main from './';

afterEach(cleanup);

test('should take a snapshot', () => {
    const { baseElement } = render(<Main />);
    expect(baseElement).toMatchSnapshot();
});

test('can have a value', () => {
    const { getByTestId } = render(<Main>Test Content</Main>);
    expect(getByTestId('main-container')).toHaveTextContent('Test Content');
});