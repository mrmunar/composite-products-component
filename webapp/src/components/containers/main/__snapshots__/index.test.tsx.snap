// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<body>
  <div>
    <div
      class="main-container"
      data-testid="main-container"
    >
      <div
        class="row justify-content-center"
      >
        <div
          class="col-lg-6"
        >
          <div
            class="text-left"
          />
        </div>
      </div>
    </div>
  </div>
</body>
`;
